flink-ml-lib/src/main/java/org/apache/flink/ml/common/param/HasFeaturesCol.java

```diff
@@ -24,6 +24,14 @@
 import org.apache.flink.ml.param.WithParams;
 
 /** Interface for the shared featuresCol param. */
+
+/**
+ * Interface for the shared featuresCol param.
```

Comment:
Should we remove the redundant description above?

And it might be simpler and softer to use the following doc:

```
/**
 * Interface for the shared featuresCol param.
 *
 * {@link HasFeaturesCol} is typically used for {@link Stage}s that implement {@link HasLabelCol}. It is preferred
 * to use {@link HasInputCol} for other cases.
 */
```

Please also update the corresponding python doc if we make the change here.

Reply:
Thanks for the review, I updated the docs of Java and Python according to your expression.